bow/694c20a0-9b96-4444-80ae-f2fa5bd1395b.md

```diff
@@ -0,0 +1,32 @@
+# bow
```

Comment:
Why is this 3 times?
They all have the same description and version 1.0.0 but different sizes :-/

Reply:
```
We hit `scipy.sparse` limits while trying to merge sparse matrices for all bags, so this is only one of three `BOW` model holding bags.
```

From the description